### ISSUE TITLE
[Enhancement]: Random play

### ISSUE THREAD

**Issue by gavinorland** [advplyr/audiobookshelf#3167]
### Type of Enhancement

None

### Describe the Feature/Enhancement

A dice button or or such-like would randomly select the next (unheard) audiobook you'll listen to and begin playback.

### Why would this be helpful?

It can be nice to have a title auto-selected (a feature also seen, for example, on Plex).

### Future Implementation (Screenshot)

Perhaps the button would go up on the top right alongside the others here for the web UI - somewhere similar for mobile.

<img width="292" alt="Menu bar buttons" src="https://github.com/user-attachments/assets/413db2c6-77d7-46e5-86f6-9edb3ea8c240">

### Audiobookshelf Server Version

2.11.0

### Current Implementation (Screenshot)

n/a

**Comment by DDriggs00** [comment]
Doesn't the "Discover" section of the library page already do this?

**Comment by gavinorland** [comment]
For me, "Discover" is a section of the Home page, not the Library page. It presents ten (apparently randomly selected) unheard audiobooks. Random play would instead, as described, directly choose only one unheard audiobook, randomly, and begin playback, all from the one button.

**Comment by nichwall** [comment]
Yes, Discover is a shelf that randomly picks 10 books that are not started and the first book of the series (if in a series) so you're not starting a series in the middle.

You can press play on the cover of the first book in the shelf and that is still "one button press" and be random, and then if you don't like it go to the next book down the list.

**Comment by gavinorland** [comment]
Yes - as such, "Discover" is not quite the same as the random play I describe. With random play, of course, there's an element of surprise. The system finds a book for you and begins playback. With "Discover" you are in a sense, still choosing. It's just an idea for the maintainer to consider.
